### DISCUSSION TITLE
Scrollbar is hard to use

### DISCUSSION THREAD

**Discussion by gitraghuram** [super-productivity/super-productivity#3098] [Q&A] [answered]
In the windows desktop, the screen for Today that is showing a list of tasks,
 is having such a thin scroll bar that it is a hit or miss for me go grab it for scrolling. 
Such precise mouse control is hard for me.
Please let me know how to make it broader.

**Comment by johannesjo** [comment, marked as answer]
If you have a mousewheel on your mouse or a touchpad you can also use those for scrolling. This is why most apps these days have very small scrollbars.

**Comment by gitraghuram** [comment]
I don't have either of them.
It looks like i can use up down arrows, pgup, pgDn buttons.
Thanks for replying.
